### PR TITLE
Post DR test documentation updates

### PR DESCRIPTION
**Pull request by smneal**
## Context

These updates come from a review of the DR process against QA

## Changes proposed in this pull request

Minor DR documentation updates

## Guidance to review

Review the updated markdown file

## Link to Trello card

https://trello.com/c/RFQySePB

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by smneal** [comment]
[Apply: simulate loss of database instance](https://trello.com/c/RFQySePB/443-apply-simulate-loss-of-database-instance)

File: docs/disaster-recovery.md [hunk 1]
```diff
@@ -45,14 +63,14 @@ In case the database instance is lost, the objectives are:
 ### Recreate the lost postgres database instance
 
 Please note, this process should take about 25 mins* to complete. In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
-First make sure it is fully gone by running
+First make sure it is fully gone by running. A manual deletion of the service requires the keys to be deleted first `cf service-keys apply-postgres-qa` then `cf delete-service-key apply-postgres-qa <service-key-namee>`.
```
Comment (RMcVelia) [inline docs/disaster-recovery.md:66]: check sentence?

File: docs/disaster-recovery.md [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
 ### Local Dependencies
 
 You will need the following tools installed to successfully complete the process:
 - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 - [cf](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
 - [conduit](https://github.com/alphagov/paas-cf-conduit)
-- pg_dump: `sudo apt-get install postgresql-client`
+- pg_dump:
+  - WSL/Linux: `sudo apt-get install postgresql-client`
+  - MacOS: `brew install postgresql@11` then ensure pg_dump is available in the PATH run `echo 'export PATH="/opt/homebrew/opt/postgresql@11/bin:$PATH"' >> ~/.zshrc` and `/opt/homebrew/opt/postgresql@11/bin/postgres -D /opt/homebrew/var/postgresql@11` if you don't want/need it is a background service.
```
Comment (RMcVelia) [inline docs/disaster-recovery.md:19]: as a background service

File: docs/disaster-recovery.md [hunk 3]
```diff
@@ -44,15 +62,16 @@ In case the database instance is lost, the objectives are:
 
 ### Recreate the lost postgres database instance
 
-Please note, this process should take about 25 mins* to complete. In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
-First make sure it is fully gone by running
+Please note, this process should take about 25 mins* to complete. In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it. When testing the process a manual deletion of the database service requires the associated keys to be deleted first. Run `cf service-keys apply-postgres-qa` to list the service keys and then `cf delete-service-key apply-postgres-qa <service-key-namee>` for each key to delete it.
```
Comment (RMcVelia) [inline docs/disaster-recovery.md:65]: service-key-namee

**Review by RMcVelia — APPROVED** [review]

**Review by tomas-stefano — APPROVED** [review]
Looks good! Thanks for leading on this 🚀

**Pull request merged**